src/main/java/com/miu/onlinemarketplace/entities/Product.java

```diff
@@ -5,8 +5,6 @@
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
-import java.util.List;
```

Comment:
You need to import java.util.cuncurrent as well

Reply:
I have imported the java.util.*.